tests/test_response_model_as_return_annotation.py

```diff
@@ -0,0 +1,141 @@
+from fastapi import FastAPI
+from fastapi.testclient import TestClient
+from pydantic import BaseModel
+
+
+class ModelOne(BaseModel):
+    name: str
+
+
+class ModelTwo(BaseModel):
+    surname: str
+
+
+app = FastAPI()
+
+
+@app.get("/valid1")
+def valid1() -> ModelOne:
+    return ModelOne(name="Test")
+
+
+@app.get("/valid2", response_model=ModelTwo)
+def valid2():
+    return ModelTwo(surname="Test")
+
+
+@app.get("/valid3", response_model=ModelTwo)
+def valid3() -> ModelOne:
+    return ModelTwo(surname="Test")
```

Comment:
```suggestion
```

This is same as the test above in lines 22-24.

Reply:
@iudeen They actually different. On lines 22-44 there is no return type annotation but 27-29 has it.